src/applications/pensions/config/chapters/04-household-information/dependentChildrenPages.js

```diff
@@ -0,0 +1,423 @@
+import merge from 'lodash/merge';
+import get from 'platform/utilities/data/get';
+
+import {
+  arrayBuilderItemFirstPageTitleUI,
+  arrayBuilderItemSubsequentPageTitleUI,
+  arrayBuilderYesNoSchema,
+  arrayBuilderYesNoUI,
+  addressUI,
+  addressSchema,
+  dateOfBirthUI,
+  dateOfBirthSchema,
+  fullNameUI,
+  fullNameSchema,
+  radioUI,
+  radioSchema,
+  ssnUI,
+  ssnSchema,
+  yesNoUI,
+  yesNoSchema,
+} from '~/platform/forms-system/src/js/web-component-patterns';
+import {
+  VaCheckboxField,
+  VaTextInputField,
+} from 'platform/forms-system/src/js/web-component-fields';
+import currencyUI from 'platform/forms-system/src/js/definitions/currency';
+import { arrayBuilderPages } from '~/platform/forms-system/src/js/patterns/array-builder';
+import {
+  DisabilityDocsAlert,
+  SchoolAttendanceAlert,
+} from '../../../components/FormAlerts';
+import { childRelationshipLabels } from '../../../labels';
+import { isBetween18And23 } from './helpers';
+import {
+  DependentSeriouslyDisabledDescription,
+  formatFullName,
+  showDependentsMultiplePage,
+} from '../../../helpers';
+
+/** @type {ArrayBuilderOptions} */
+const options = {
+  arrayPath: 'dependents',
+  nounSingular: 'dependent child',
+  nounPlural: 'dependent children',
+  required: false,
+  isItemIncomplete: item =>
```

Comment:
oh wow! lots of things to check. maybe from the pattern side, we can consider making it so you don't need to include this option in the future, so it can just scan all the required fields.